tests/com/reason/esy/EsyProcessTest.java

```diff
@@ -0,0 +1,39 @@
+package com.reason.esy;
+
+import com.intellij.openapi.project.Project;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+public class EsyProcessTest {
+
+  @Mock
+  private Project mockProject;
```

Comment:
Can you extend the test from BasePlatformTestCase ? you'll get access to a getProject() function, and then mockito dep might not be needed

Reply:
Sure, I'll give that a shot. I'm planning on extending that test with more cases and will likely need mockito in the future. I'll drop it until it's needed though.